service/pipeline.ml

```diff
@@ -69,7 +71,8 @@ let build_with_docker ~repo ~analysis source =
     "fedora", build (module Conf.Builder_amd3) @@ "fedora-30-ocaml-" ^ default_compiler;
     (* oraclelinux doesn't work in opam 2 yet: *)
     (* build (module Conf.Builder_amd3) @@ "oraclelinux-7-ocaml-" ^ default_compiler; *)
-    "lint", (lint_result, job_id lint_result);
+    "lint-src", (lint_fmt_result, job_id lint_fmt_result);
+    "lint-doc", (lint_doc_result, job_id lint_doc_result);
```

Comment:
Are these names presented to the user ?
Even if no, what about `Formatting` and `Documentation` ? No need to mangle "lint" in the name, every lint steps will most probably be independent and shown separately to the user (no single "lint" step doing everything).

Reply:
They are visible; I've made them clearer in https://github.com/ocurrent/ocaml-ci/pull/105/commits/674cc7df3dfe8e8e757afb5bc7e681956905d273, but ultimately I think the solution here is to have a hierarchical (or graph-based) display of jobs.

I disagree with not including 'lint' in the name. "Documentation" could mean anything from '_building documentation_' to '_linting documentation_' to '_deploying documentation to web API_' (all of which we have plans to do).

Reply:
The "Lint" prefix is unnecessary for now and can be added when an other documentation related job is added.

Reply:
As discussed in person, the "Lint" prefix is helpful in describing what this job does (and any related jobs can be considered when they arise).